### DISCUSSION TITLE
no + button to add a database

### DISCUSSION THREAD

**Discussion by IamAnjaehyun** [pawelsalawa/sqlitestudio#4730] [Q&A] [answered]
### Details
Here you describe your problem details, or proposal for enhancement.

### Steps to reproduce
This applies only to bug reports. If this is a feature request, delete this section.

### Operating system
This applies only to bug reports. If this is a feature request, delete this section.

### SQLiteStudio version
This applies only to bug reports. If this is a feature request, delete this section.
![image](https://user-images.githubusercontent.com/101981639/226280522-746203fe-d506-41bc-9c3f-3b3e16a54240.png)

There is no + button to add a database on this screen. I am not an existing user, but a first-time user, how should I solve this problem?

**Comment by pawelsalawa** [comment, marked as answer]
Recent versions of SQLiteStudio do not have 2 buttons (the "plus" and "folder") - instead there is just one button "folder" and using it you can add both new and existing databases, so just use that button.

**Reply by IamAnjaehyun** [reply]
thx !!

**Reply by stronghold1995** [reply]
Many thanks

**Reply by ghost** [reply]
tnkss
